src/tunacode/configuration/paths.py

```
@@ -138,23 +140,25 @@ def check_for_updates() -> tuple[bool, str]:
-
-            latest_version = latest_version.strip()
+            latest_version = semver.Version.parse(versions[0])
 
-            if latest_version > current_version:
-                return True, latest_version
+            if latest_version.compare(current_version) > 0:
+                return True, str(latest_version)
 
-        return False, current_version
+        return False, str(current_version)
     except Exception:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)